source/tutorials/nix-language.md

```diff
@@ -1804,16 +1804,16 @@ Explanation:
 
 To get things done:
 
-- [](declarative-reproducible-envs) – create reproducible shell environments from a Nix file
+- [Declarative and reproducible developer environments](declarative-reproducible-envs) – create reproducible shell environments from a Nix file
 - [Garbage Collection](https://nixos.org/manual/nix/stable/package-management/garbage-collection.html) – remove unused build results from the Nix store
 
 To learn more:
 
 If you worked through the examples, you will have noticed that reading the Nix language reveals the structure of the code, but does not necessarily tell what the code actually means.
 
-Often it is not possible to determine from the code at hand
-- the data type of a named value or function argument.
-- the data type a called function accepts for its argument.
+Often it is not possible to determine from the code at hand:
+- the data type of a named value or function argument,
+- the data type a called function accepts for its argument, or
```

Comment:
This is not required grammatically and makes the list harder to parse for no good reason.